src/Microsoft.Restier.EntityFramework7/Model/ModelProducer.cs

```diff
@@ -202,6 +203,29 @@ private static void SetComputedAnnotation(EdmModel model, IEdmProperty target)
             {
                 return null;
             }
+
+            if (Type.GetTypeCode(propertyType) == TypeCode.DateTime)
```

Comment:
I remember some .NET profiles have retired `Type.GetTypeCode` thus we copied the whole `TypeCode` enum into ODataLib. Is there any reason we are not using `propertyType == typeof(DateTime)` here?